data/migrations/1_create_msmt_results.sql

```diff
@@ -11,36 +13,155 @@ DROP TABLE `measurements`;
 
 CREATE TABLE `results` (
     `id` INTEGER PRIMARY KEY AUTOINCREMENT,
-    `name` VARCHAR(255),
-    `start_time` DATETIME,
-    `runtime` REAL,
-    `summary` JSON,
-    `done` TINYINT(1),
-    `country` VARCHAR(2),
-    `asn` VARCHAR(16),
-    `network_name` VARCHAR(255),
-    `data_usage_up` INTEGER,
-    `data_usage_down` INTEGER
+    -- This can be one of "websites", "im", "performance", "middlebox".
+    `test_group_name` VARCHAR(16) NOT NULL,
+    -- We use a different start_time and runtime, because we want to also have
+    -- data to measure the overhead of creating a report and other factors that
+    -- go into the test.
+    -- That is to say: `SUM(runtime) FROM measurements` will always be <=
+    -- `runtime FROM results` (most times <)
+    `start_time` DATETIME NOT NULL,
+    `runtime` REAL NOT NULL,
+
+    -- Used to indicate if the user has seen this result
+    `is_viewed` TINYINT(1) NOT NULL,
+
+    -- This is a flag used to indicate if the result is done or is currently running.
+    `is_done` TINYINT(1) NOT NULL,
+    `data_usage_up` INTEGER NOT NULL,
+    `data_usage_down` INTEGER NOT NULL,
+    -- It's probably reasonable to set the maximum length to 260 as this is the
+    -- maximum length of file paths on windows.
+    `log_file_path` VARCHAR(260) NOT NULL
```

Comment:
This is unused, can be removed.

Reply:
On desktop it's actually more convenient to have this explicit in the DB as the paths can vary depending on the platform. I am going to keep it.

Reply:
so are you going to add the report file path as well?